src/axolotl/core/trainers/base.py

```diff
@@ -143,7 +144,21 @@ def create_scheduler(
                     min_lr_ratio=self.args.cosine_min_lr_ratio,
                 )
             else:
-                return super().create_scheduler(num_training_steps, optimizer=optimizer)
+                super().create_scheduler(num_training_steps, optimizer=optimizer)
+                if self.args.jagged_restart_steps:
+                    warmup_steps = (
+                        self.args.jagged_restarts_warmup_steps or 10
+                    )
+                    anneal_steps = (
+                        self.args.jagged_restarts_anneal_steps or 1
+                    )
+                    self.lr_scheduler = JaggedLRRestartScheduler(  # pylint: disable=attribute-defined-outside-init
+                        optimizer,
+                        self.lr_scheduler,
+                        self.args.jagged_restart_steps,
+                        warmup_steps,
+                        anneal_steps,
+                    )
```

Comment:
- Do we need to get the return from `super().create_scheduler`?

- Would it make more sense to move this section one tab left as it could apply for other lr schedulers too?